__tests__/classValidatorToJsonSchema.test.ts

```diff
@@ -0,0 +1,53 @@
+import { IsString, MinLength } from 'class-validator'
+import { classValidatorToJsonSchema, validationMetadatasToSchemas } from '../src'
+
+export class User {
+  @MinLength(5)
+  @IsString()
+  name: string;
+}
```

Comment:
Can we also test that if the two User classes both have a property with the same name but different class-validator decorators, only the desired decorators get applied in the resulting schema?